app/scripts/views/confirm.js

```diff
@@ -45,6 +45,7 @@ define(function (require, exports, module) {
       var email = this.getAccount().get('email');
       var isSignIn = this.isSignIn();
       var isSignUp = this.isSignUp();
+      var openWebmailButtonVisible = this.isOpenWebmailButtonVisible(email);
```

Comment:
I was calling it below, but all the other vars were being set before. So I guessed it was some kind of convention and changed it.